subtests/docker_cli/liverestore/liverestore.py

```diff
@@ -0,0 +1,185 @@
+r"""
+Summary
+---------
+
+Test live-restore: running container should survive docker daemon restart.
+
+Operational Summary
+----------------------
+
+#. Run a docker container
+#. Restart docker daemon
+#. Verify that container is listed under 'docker ps' and is still running.
```

Comment:
(Haven't read below) How does the test prove the container was running while the deamon was not.  In other words, that restarting the daemon wasn't also responsible for starting the container (i.e. maybe the default --restart argument changes)